.github/workflows/ci.yml

```
@@ -0,0 +1,67 @@
+      with:
+        useConfigFile: true
+        updateAssemblyInfo: true
+    
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '9.0.x'
+    
+    - name: Restore dependencies
+      run: dotnet restore
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)